e2e-tests/tests/undefined-menu-button.spec.ts

```diff
@@ -0,0 +1,8 @@
+import { expect } from "@playwright/test";
+import { studioTest } from "./infra/studioTest.js";
+
+studioTest("renders undefined menu button", async ({ page, studioPage }) => {
+  await studioPage.setActiveComponent("Banner");
+  await page.getByLabel("Toggle undefined value menu").first().click();
+  await expect(page).toHaveScreenshot();
```

Comment:
can you add a screenshots to check the that if you scroll to the right, you're able to see the whole menu text? ideally we'd also check that if you scroll down, the menu remains next to the button, but that might be hard to implement with our current setup until we have more realistic test pages